dockhand-lru-registry/values.yaml

```diff
@@ -0,0 +1,49 @@
+registry:
+  image:
+      repository: registry
+      pullPolicy: IfNotPresent
+      tag: 2.7.1
+  cache:
+    enabled: true
+    emptyDir:
+      sizeLimitSoft: 10Gi
+      sizeLimitHard: 20Gi
+    persistence:
+      enabled: true
+      #storageClass: "-"
+      accessMode: ReadWriteOnce
+      size: 100Gi
+
+proxy:
+  debug: false
+  cleanSettings:
+    targetUsagePercentage: 80.0
+    cleanTagPercentage: 10.0
+    cron: "0 0 * * *"
+    timezone: "GMT"
```

Comment:
I would expect the default timezone to be the system time, not GMT.  So we should leave this blank and the library should not attempt to set a timezone if one is not selected

Reply:
Switched to use "Local"